src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/DocumentResource.kt

```diff
@@ -98,13 +100,19 @@ class DocumentResource(private val documentService: DocumentService) {
   )
   fun getDocumentIds(
     @Schema(description = "The booking id") @PathVariable bookingId: Long,
-    @Schema(description = "The template name") @PathVariable templateName: String,
-  ) = documentService.findAllIds(bookingId, templateName)
+    @RequestParam(value = "templateName", required = true)
+    @Parameter(
+      description = "The unique name of the template used for a document",
```

Comment:
this confused me when I first saw it, might be worth adding that this parameter can be repeated.  Would be interested to see what swagger is generated for it.

Reply:
good idea